documentation/docs/20-core-concepts/30-form-actions.md

```diff
@@ -178,8 +178,20 @@ export const actions = {
 
 ```diff
 /// file: src/routes/login/+page.svelte
+<script>
+	import { page } from '$app/stores'
+
+	/** @type {import('./$types').ActionData} */
+	export let form
+</script>
+
 <form method="POST" action="?/login">
 -	<input name="email" type="email">
++	{#if form?.incorrect && $page.status === 400}
++		<p>There are errors in your form submission. See below</p>
++	{:else if form?.incorrect && $page.status === 409}
++		<p>Account already exists, please log in or register with a different email.</p>
```

Comment:
While the example is less contrived now, it's still not ideal since
- you can make that distinction through returning a special boolean from `return invalid(..)`, which would save some keystrokes
- it's insecure (we had such a version previously, where it was pointed out that telling the user that the email exists can be used for further attacks)

I think the best way to handle this is probably to keep this out of the code example and instead note about this in a separate paragraph below. Something like "The status code returned from `invalid` is available through `$page.status`"